src/components/memory/view/memory.html

```diff
@@ -0,0 +1,27 @@
+<div class="container-fluid">
+  <div class="row">
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <h3 class="panel-title">Memory</h3>
+      </div>
+      <div class="panel-body">
+        <div class="form-horizontal" ng-if="!$ctrl.isMemoryDefined()">
+          <div class="form-group">
+            <label class="col-md-8 control-label">Memory management has not been defined. Click
+              <button ng-click = "$ctrl.createDefaultMemory()"> here</button> to define it</label>
```

Comment:
I do like this message, however what does a user do if they want to remove the memory configuration on an existing cache/template? With stores/loaders they can simply select the No option in the combo box and the previously configured store/loader is removed on update.  Obviously we would need to add this logic to the CacheConfigService as well.